### DISCUSSION TITLE
Incognito mode - how private is it?

### DISCUSSION THREAD

**Discussion by dsommers** [Eloston/ungoogled-chromium#2119] [General]
I'm reading this little post here:  https://futurism.com/the-byte/google-engineers-joked-incognito-mode

How does the Ungoogle Chromium fare by comparison to the upstream Chromium and Chrome browsers?

**Comment by networkException** [comment]
Incognito mode does not hide your activity from websites you visit. It is trivial for any website to recognize you as the same user when using incognito mode. ungoogled-chromium has limited fingerprinting protection mechanisms which MAY block some fingerprinting attempts, this however all falls apart if a website uses behavior tracking or simply matches against your IP address.

ungoogled-chromium does provide an advantage over vanilla chromium / Google Chrome in terms of blocking background requests to google servers (for example by safebrowsing)

**Comment by tomasz1986** [comment]
The main purpose and/or use case of incognito is not to store any browsing history on your local device, not to grant any special privacy protection online. If you need anonymity, then something like Tor Browser should be more in line with that.

**Comment by dsommers** [comment]
Okay, so to clarify a bit ... I don't expect Incognito mode to make my online surfing anonymous or hide my activity on a site.  That I consider out-of-scope for a browser to tackle.  But it's more the part I've highlighted from the article:

> Google, in its defense, argues that it makes it clear to users that **Incognito mode isn't fully private, and that its users have already consented to have their data tracked by the company.** 

That is the troublesome aspects, that Ingognito mode in Chrome/Chromium still sends tracking data to Google.

The Bloomberg article provides a lot more details: https://www.bloomberg.com/news/articles/2022-10-11/google-s-incognito-inspires-staff-jokes-and-a-big-lawsuit

> Data on where Incognito Mode users go online, what they do on certain sites and what ads they view could be used for conversion tracking, which lets advertisers learn how users interact with ads, according to Egelman.

And even more on this issue: https://www.bloomberg.com/news/articles/2021-02-26/google-judge-disturbed-that-even-incognito-users-are-tracked

**Reply by tomasz1986** [reply]
Yeah, so in this aspect, you should be better off with Ungoogled Chromium, as in contrary to Chrome/Chromium proper, the browser does not send anything to Google. Websites themselves can still track you, of course, and you should also be aware of what extensions you install too.
